servers/src/main/java/tachyon/worker/block/allocator/Allocator.java

```diff
@@ -25,11 +31,32 @@
  * Interface for the allocation policy of Tachyon managed data.
  */
 public interface Allocator {
+
+  class Factory {
+    /**
+     * Create a new {@link Allocator} instance
```

Comment:
Could you javadoc here that, the returned `Allocator` is `MaxFreeAllocator` by default.